src/EA.Weee.RequestHandlers/Admin/Obligations/GetPcsObligationsCsvHandler.cs

```diff
@@ -56,7 +56,7 @@ public async Task<CSVFileData> HandleAsync(GetPcsObligationsCsv message)
 
             var fileContent = csvWriter.Write(authority.Schemes);
             var timestamp = DateTime.Now;
-            var fileName = $"{authority.Abbreviation}_pcsobligationuploadtemplate{timestamp.ToString("dd/MM/yyyy_HHmm")}.csv";
+            var fileName = $"{authority.Abbreviation}_pcsobligationuploadtemplate{timestamp.ToString("_yyyy_dd/MM/yyyy_HHmm")}.csv";
```

Comment:
is there a constant class for keeping date and time formats such as this one?
if there is one, would you mind adding it in?

Reply:
@NateHogan Meg makes a good point if you could do this change in your test branch... i.e. put _yyyy_dd/MM/yyyy_HHmm into a constant somewhere

Reply:
Good idea, moved the DateTimeConstants class to the core project, and added the format as a constant